modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/CalciteQueryProcessorTest.java

```diff
@@ -99,26 +99,28 @@ public void testCountWithJoin() throws Exception {
         Map<Integer, RISK> mRisk = new HashMap<>(65000);
 
         for (int i = 0; i < 65000; i++)
-            mRisk.put(1, new RISK(i));
+            mRisk.put(i, new RISK(i));
 
         RISK.putAll(mRisk);
 
         Map<Integer, TRADE> mTrade = new HashMap<>(200);
 
         for (int i = 0; i < 200; i++)
-            mTrade.put(1, new TRADE(i));
+            mTrade.put(i, new TRADE(i));
 
         TRADE.putAll(mTrade);
 
         for (int i = 0; i < 80; i++)
-            BATCH.put(1, new BATCH(i));
+            BATCH.put(i, new BATCH(i));
 
         awaitPartitionMapExchange(true, true, null);
 
-        QueryEngine engine = Commons.lookupComponent(grid(1).context(), QueryEngine.class);
+        QueryEngine engine = Commons.lookupComponent(ignite.context(), QueryEngine.class);
```

Comment:
hope we need to extend this and lower tests, append additional call from server nodes and check results for equality.

Reply:
think it should be done under separate ticket